src/Akeneo/Connectivity/Connection/back/Application/Webhook/WebhookEventBuilder.php

```diff
@@ -49,10 +43,11 @@ public function build(BulkEventInterface $pimEventBulk, array $context = []): ar
 
         /** @var UserInterface $user */
         $user = $context['user'];
+        $isUsingUuid = $context['is_using_uuid'] ?? false;
 
         $eventDataCollection = $eventDataBuilder->build(
             $pimEventBulk,
-            new Context($user->getUsername(), $user->getId())
+            new Context($user->getUsername(), $user->getId(), $isUsingUuid)
```

Comment:
You set it required and bool in the resolver.
```suggestion
            new Context($user->getUsername(), $user->getId(), $context['is_using_uuid'])
```